<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Password Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="username_input"/>
    </item>
    <item>
     <widget class="QLineEdit" name="password_input">
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="add_button">
      <property name="text">
       <string>Add Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="view_button">
      <property name="text">
       <string>View Passwords</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_button">
      <property name="text">
       <string>Delete Password</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="password_display">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
